site/app/templates/admin/Report.twig

```diff
@@ -3,13 +3,37 @@
         $(document).ready(function() {
             // Bind click listener to grade summaries button
             $('#grade-summaries-button').click(function() {
-                $('#grade-summaries-last-run').html('Running...');
+                $('.grade-summaries-last-run').html('Running...');
+            });
+            // Bind click listener to download buttons
+            $('.download-btn').click(function() {
+                let path = $(this).data('path');
+                let dir = $(this).data('dir');
+                downloadFile(path, dir);
             });
         });
     </script>
 
     <div id="grade-report">
         <h1>Grade Reports</h1>
+        <div id="rainbow-grade-report-csv" class="card">
+            <div class="title-cont">
+                Pushing this button will generate a CSV file (for download) of Rainbow Grades.
+            </div>
+            <div class="card-btn-cont">
+                <button class="btn btn-primary key_to_click download-btn"
+                        data-path="/var/local/submitty/courses/f23/development/rainbow_grades/output.csv"
+                        data-dir="rainbow_grades"
```

Comment:
These are hardcoded values. path should be dynamically generated for the specific semester/course context we are in. Also dir is completely static so that could be moved entirely to the JS.

Reply:
I apologize for the oversight